### ISSUE TITLE
Creating product with categories returns an error "Invalid JSON returned"

### ISSUE THREAD

**Issue by danielemesh** [woothemes/wc-api-php#17]
WC v2.5.5
API v3

I have a problem creating a new product with categories.
I am using the example from the API docs as follows:

```
$data = [
    'product' => [
        'title' => 'Premium Quality',
        'type' => 'simple',
        'regular_price' => '21.99',
        'description' => 'Pellentesque',
        'short_description' => 'Pellentesque habitant',
        'categories' => [
            64,
            193
        ]
    ]
];

print_r($woocommerce->post('products', $data));
```

The given categories ids exists, and still I get an error. 
The weird thing is that the product gets created even though an exception is thrown.

The error:

```
Automattic\WooCommerce\HttpClient\HttpClientException: Invalid JSON returned in /Users/daniel/Sites/outsiders/wp-content/themes/outsiders-theme/lib/vendor/automattic/woocommerce/src/WooCommerce/HttpClient/HttpClient.php on line 303
```

When I try to create the product without the categories array, it works fine.
After trying all sorts of manipulations, I realized there is a problem to pass an array inside the $data variable.
Because when I tried adding the images array, it also threw an exception, but still created the product with the images.
I also tried an empty categories array, empty images array etc, and whenever I pass an array, the exception is thrown...

Any idea??

**Comment by claudiosmweb** [comment]
Fixed in 1.0.2. Update before open issues.

**Comment by danielemesh** [comment]
I have the last version v1.0.2
I also copied the changes from #12 cause for some reason it wasn't included when I installed with composer.
I wouldn't open an issue for no reason.
